sdk/iot/Azure.Iot.Hub.Service/src/JobsClient.cs

```diff
@@ -3,23 +3,205 @@
 
 using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Iot.Hub.Service.Models;
 
 namespace Azure.Iot.Hub.Service
 {
     /// <summary>
-    /// Jobs Client place holder
+    /// Jobs client to support Import,Export and Scheduled Jobs.
     /// </summary>
     public class JobsClient
     {
+        private readonly JobsRestClient _jobsRestClient;
+
+        protected JobsClient()
+        {
+            // This constructor only exists for mocking purposes.
```

Comment:
Should this be a doc comment?

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs#L93

Reply:
I looked at other clients we have and changed this to doc comments with comments similar to what we have in other clients.